types/classnames/classnames-tests.ts

```diff
@@ -1,8 +1,10 @@
 import classNames = require('classnames');
 import * as classNames2 from 'classnames';
+import classNames3 from 'classnames/esmodule';
```

Comment:
I'm a little confused -- this `/esmodule` sub-module doesn't seem to exist in classnames. Wouldn't these typings mislead you into importing a nonexistent file?